interface/globals.php

```diff
@@ -4,6 +4,7 @@
 require_once(dirname(__FILE__) . "/../common/compatibility/Checker.php");
 
 use OpenEMR\Checker;
+use OpenEMR\Core\Header;
```

Comment:
Maybe.... probably. But IDE integration will probably lack w/o that statement. And yes, we definitely want to break out the twig stuff (Don't worry, I'm definitely working on it, most of the puzzle pieces are in place :D )